metadata/hostmetadata/host.go

```diff
@@ -122,9 +124,11 @@ func int8ArrayToByteArray(in []int8) []byte {
 }
 
 // GetOS returns a struct with information about the host os
-func GetOS() (info *OS, err error) {
-	info = &OS{}
-	hInfo, err := hostInfo()
+func GetOS() (*OS, error) {
+	var (
```

Comment:
this was required?

Reply:
it was complaining about shadow error so i had to create a var hInfo and if I do that it gofumtd would have complained.. So just merged all var into one declaration